src/libcore/alloc.rs

```diff
@@ -169,7 +169,7 @@ impl Layout {
     /// Panics if the combination of `self.size()` and the given `align`
     /// violates the conditions listed in
     /// [`Layout::from_size_align`](#method.from_size_align).
-    #[unstable(feature = "allocator_api", issue = "32838")]
+    #[stable(feature = "alloc_layout_extra", since = "1.31.0")]
```

Comment:
Hmm... will this make it in time? Hasn't 1.31 gone into beta yet?

Reply:
I just used the version number of the current latest nightly, which is 1.31.

Reply:
cc @pietroalbini ^

Reply:
1.31 will go into beta in the weekend.

Reply:
FCP takes 10 days, so if merging needs to wait on FCP, this will be 1.32.